pkg/certificates/certificates_test.go

```diff
@@ -609,7 +609,7 @@ func TestCertificateSigner_Sign_E2E(t *testing.T) {
 				require.NoError(t, err)
 				require.NotNil(t, serverOpts)
 
-				service, err := server.New(serverOpts, sc.Address, sc.PrivateAddress, nil, config)
+				service, err := server.New(zap.NewNop(), serverOpts, sc.Address, sc.PrivateAddress, nil, config)
```

Comment:
Why do we have the NewNop() Logger here?

Reply:
fixed